### PR TITLE
[api/frontend] Add failure handler and error message

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Related issues

* https://github.com/OpenEx-Platform/openex/issues/357

### CODE REVIEW

**Comment by RomuDeuxfois** [comment]
Use snackbar without duration for error message

File: openex-api/src/main/java/io/openex/config/AppSecurityConfig.java [hunk 1]
```diff
@@ -221,17 +229,23 @@ public User userManagement(String emailAttribute, String registrationId, List<St
...
-  public User userOauth2Management(OAuth2AccessToken accessToken, ClientRegistration clientRegistration,
+  public User userOauth2Management(
+      OAuth2AccessToken accessToken,
+      ClientRegistration clientRegistration,
       OAuth2User user) {
     String emailAttribute = user.getAttribute("email");
     String registrationId = clientRegistration.getRegistrationId();
     List<String> rolesFromToken = extractRolesFromToken(accessToken, registrationId);
+    if (isBlank(emailAttribute)) {
+      OAuth2Error authError = new OAuth2Error("invalid_configuration", "You need a public email in your github account", "");
```
Comment (guillaumejparis) [inline openex-api/src/main/java/io/openex/config/AppSecurityConfig.java:246]:
```suggestion
      OAuth2Error authError = new OAuth2Error("invalid_configuration", "You probably need a public email in your " + 
registrationId + " account", "");
```

File: openex-front/src/components/ThemeDark.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ export default (
     mode: 'dark',
     primary: { main: primary || '#00b1ff' },
     secondary: { main: secondary || '#ec407a' },
+    error: { main: '#f44336' },
```
Comment (guillaumejparis) [inline openex-front/src/components/ThemeDark.js:15]: Not needed anymore ? 
```suggestion
```

**Review by guillaumejparis — APPROVED** [review]
Please check the two comments otherwise LGTM

File: openex-front/src/components/ThemeDark.js (hunk 2 above)
Comment (RomuDeuxfois) [reply openex-front/src/components/ThemeDark.js:15]: Can be usefull for the next development

File: openex-api/src/main/java/io/openex/config/AppSecurityConfig.java (hunk 1 above)
Comment (RomuDeuxfois) [reply openex-api/src/main/java/io/openex/config/AppSecurityConfig.java:246]: Good catch
And I forget I18n

Comment (guillaumejparis) [reply openex-api/src/main/java/io/openex/config/AppSecurityConfig.java:246]: Oh ! Right

**Comment by RomuDeuxfois** [comment]
PR updated and ready for a second round

**Pull request merged**